pytorchfi/core.py

```diff
@@ -381,3 +381,49 @@ def get_fmaps_W(self, layer):
 
     def get_fmap_HW(self, layer):
         return (self.get_fmaps_H(layer), self.get_fmaps_W(layer))
+
+    def print_pytorchfi_layer_summary(self):
```

Comment:
for now i think this is fine, but hopefully we can move this to another file to slim down size of core.py in the future